# KOLD dataset

Kold dataset is created by School of Computing in KAIST and SoftlyAI Research as part of a research reportin 2022. The report and the Github could be found below.

- Research Paper: https://aclanthology.org/2022.emnlp-main.744/
- Github: https://github.com/boychaboy/KOLD/tree/main

In [ ]:
import pandas as pd 

dataset = pd.read_json('kold_v1.json') # Read json file in pandas


In [6]:
dataset.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 40429 entries, 0 to 40428
Data columns (total 11 columns):
 #   Column      Non-Null Count  Dtype         
---  ------      --------------  -----         
 0   guid        40429 non-null  object        
 1   source      40429 non-null  object        
 2   date        40429 non-null  datetime64[ns]
 3   title       40429 non-null  object        
 4   comment     40429 non-null  object        
 5   OFF         40429 non-null  bool          
 6   TGT         20310 non-null  object        
 7   GRP         12413 non-null  object        
 8   OFF_span    40429 non-null  object        
 9   TGT_span    40429 non-null  object        
 10  raw_labels  40429 non-null  object        
dtypes: bool(1), datetime64[ns](1), object(9)
memory usage: 3.1+ MB


Kold dataset has total 40429 rows with 11 different columns that contain source, date, title, comment and the detailed information about it being hate speech or not.

In [7]:
dataset.head()

,guid,source,date,title,comment,OFF,TGT,GRP,OFF_span,TGT_span,raw_labels
0,kold-v1_00000,naver_news,2021-07-25,페미니즘이 범죄가 되는 나라 [삶과 문화],남녀평등 주장할 거면 여성징병제에도 동의하라고ㅋㅋㅋ 그리고 내 말에 그냥 시비만 걸...,True,group,others-feminist,쳐받으면서 왜 차별받는다고 말하는지 말해보라고ㅋㅋㅋ,,"[{'offensiveness': True, 'annotator_id': 19151..."
1,kold-v1_00001,naver_news,2021-04-13,[젠더의학①] 여성은 ‘몸집 작은 남성’이 아니다,의학에도 젠더 사상이 붙네 ㄷㄷ;,True,untargeted,None,젠더 사상이 붙네 ㄷㄷ;,,"[{'offensiveness': False, 'annotator_id': 1856..."
2,kold-v1_00002,naver_news,2021-09-15,“책 생명 늘려야죠”… 문학 속 ‘성차별’ 패치 떼는 출판계,"루브르 박물관에 있는 모나리자 머리도 단발로 수정하고,미국에 있는 자유의 여신상도 ...",True,group,others-feminist,"모나리자 머리도 단발로 수정하고,미국에 있는 자유의 여신상도 콘크리트 부어서 가슴부...",,"[{'offensiveness': True, 'annotator_id': 18177..."
3,kold-v1_00003,naver_news,2021-01-14,이루다로 촉발된 젠더 논쟁... 개인정보 유출이란 본질 외면,진짜 어이가 없네 딥페이크 만든 사람들도 처벌하고 알페스 만든 사람들도 처벌하라니깐...,True,untargeted,None,왜 쫄리나..?,,"[{'offensiveness': True, 'annotator_id': 16616..."
4,kold-v1_00004,naver_news,2021-05-06,"GS25, 브레이브걸스 포스터 또 젠더 이슈 논란",브레이브걸스=페미아이돌,True,group,others-others,페미아이돌,브레이브걸스,"[{'offensiveness': True, 'annotator_id': 18808..."
